lib/ometajs/core.js

```diff
@@ -548,24 +569,23 @@
 		_or: function() {
```

Comment:
I see very large, consistent improvements with this version of `_or` over the previous version of ~25% for my test case on https://github.com/resin-io-modules/odata-parser/blob/cb57ff359144a8630a6bf584ae05f25c74a13665/odata-parser.js which is fairly heavy on `_or` calls, and lesser but still significant improvements for https://github.com/resin-io-modules/sbvr-parser/blob/026f52be953daf52620ef7e5213279d02b7a9dee/sbvr-parser.js which is less heavy on `_or` calls